settings/barndoors.cmd

```diff
@@ -0,0 +1,16 @@
+############ BARNDOORS ##############
+# Create a barndoor soft motor record on top of a normal motor record which allow the gap to be set for the motor
+# Copy this file to C:\Instrument\Settings\config\<machine>\configurations\galil
+
+$(IFDMC01) epicsEnvSet "CALIB_DIR" "C:/Instrument/Settings/calib/barndoors"
+
+# Define a barndoors from on first controller (if present)
```

Comment:
What? I can't make sense of this comment. I think there might be an unnecessary from...